### ISSUE TITLE
Language flag is missing for MWCC

### ISSUE THREAD

**Issue by tomsons26** [decompme/decomp.me#913]
Language flag that specifies if the target is cpp or c isn't specified for MWCC so language server is stuck in c mode.
I don't work with this compiler so don't have suggestions, just making a issue so it's not forgotten.

**Comment by ethteck** [comment]
@tomsons26 can you rephrase this? I'm not following. We have the language flag for MWCC already. Is the issue just that the setting of this flag isn't used by the language server? In which case this is a frontend bug and not a backend one.

Am I correct in my understanding?

**Comment by ethteck** [comment]
https://github.com/decompme/decomp.me/blob/main/backend/coreapp/serializers.py#L224 okay so I think the issue is probably that this method is broken. We should probably write some unit tests to confirm it works. If it does work, then some more investigation will be necessary.
